src/api/tigl.cpp

```diff
@@ -1,8 +1,8 @@
-/* 
+/*
```

Comment:
There are many style changes in the tigl.cpp file. Please remove all style changes and commit only the new function.